kronosnet.spec.in

```diff
@@ -506,6 +507,10 @@ Requires: libknet1-crypto-plugins-all
 
 %files -n libknet1-plugins-all
 
+%if %{with debuginfo}
+%debug_package
```

Comment:
shouldn't this be set to 1 ? I am not sure, so I take your word for it.

Reply:
@fabbione 
did you mean _enable_debug_packages?
it is only available on redhat distrbutions.

%debug_package is a function, which is available both on redhat distributions and SUSE distributions. 

and by the way, I just found one issue, fedora will by default set _enable_debug_packages to 1, which will build debuginfo package, while openSUSE does not. when use --enable-rpm-debuginfo, fedora will raise an error that %package debuginfo is defined.

There are two methods to deal with this error:
1. just follow the distrubutions' default behavior. If the system has enabled debuginfo by default like fedora, kronosnet will always build the debuginfo package(s)
2. use --enable-rpm-debuginfo to control this, undefine _enable_debug_packages first, then use  the rpm-debuginfo flag to control whether to build the package.

which do you prefer?

Reply:
Just to recap from IRC:

11:54 < fabbione> - don't specify anything == system defaults
11:54 < fabbione> - specify will undefine and override with new value based on enable/disable state